upgrade/README.md

```diff
@@ -0,0 +1,26 @@
+### How to run upgrade scripts
+
+Install dependencies in root project and compile the contracts
+```bash
+cd ..
+npm i
+npm run compile
+```
+
+Install dependencies from `upgrade` folder
+```bash
+npm i
```

Comment:
should go the `upgrade` directory first, right?

Reply:
Thanks, fixed https://github.com/poanetwork/tokenbridge-contracts/pull/337/commits/654048c037eed41f6bd689d2dc44099ca00ef7ef